### DISCUSSION TITLE
CReating a user with CLI

### DISCUSSION THREAD

**Discussion by cmorgankpf** [community/community#121152] [Enterprise]
i have the UI up and running and i deleted the admin user name and created a new one. but the server rebooted over night due to a VM issue and i forgot to get the password and username written down.
Can i create a user throught he CLI?
If so, how do i get the cli to run?
tried the sudo python3 -m FreeTAKServer.views.CLI and could not get it to run
I get an error /usr/bin/python3: error while finding module specification for FreeTAKServer.views.CLI(module not found)
I am new to this TAK server and just trying to get it to work

**Comment by ZFPuhi** [comment]
Hello,

The master password is set in the SQLite DB, you can use a SQLite viewer to see it or delete the database and start over. Also if you have done the wizard installation for the FTS , then the default location is located in the '/opt'.

The above is for getting in as you have forgotten the new username/password.
